README.md

```diff
@@ -16,17 +16,30 @@ Install go dep
 go get -u github.com/golang/dep/cmd/dep
 ```
 
+Install atlas-db-controller for managing database resources:
+
+  * To create atlas-db custom resource and controller follow [link](https://github.com/infobloxopen/atlas-db/blob/master/README.md).
+  * To create database, tables and schema resources locally to be used by contacts app, modify `contacts-localdb.yaml` following specifications mentioned in [link](https://github.com/infobloxopen/atlas-db/blob/master/README.md) and run:
+  ```
+    make db-up
+  ```
+  * To use RDS database, tables and schema resources used by contacts app, modify `contacts-rds.yaml` and run"
```

Comment:
tables and schema are same not 2 different resources, can you reword it. like 

To use RDS database instance, database, schema/tables resources ...